onetimepass/otp.py

```diff
@@ -296,11 +320,100 @@ def get_alias_from_uri(uri: str) -> AliasSchema:
         params=uri_parsed.params,
     )
 
+    db = JSONEncryptedDB(path=settings.DB_PATH, key=keyring_get().encode())
+    data = get_db_data(db)
+    data.add_alias(alias, alias_data)
+    db.write(data)
+    if not quiet:
+        click.echo(f"{alias} added")
+
 
-@otp.command(help="Add the new secret as the specified ALIAS.")
+@add.command("hotp")
```

Comment:
I would suggest adding `help="Add the new HOTP secret as the specified ALIAS."`, WDYT?